managers/oclc_catalog.py

```diff
@@ -102,7 +103,7 @@ def _get_other_editions(self, oclc_number: int, offset: int=0):
 
             status_code = other_editions_response.status_code
 
-            if other_editions_response.status_code != 200:
+            if other_editions_response.status_code not in self.OK_STATUS_CODES:
```

Comment:
is there a reason why we shouldn't just check `other_editions_response.ok`?

Reply:
you make a good point!